Sources/CacheAdvance/CacheAdvance.swift

```diff
@@ -57,8 +57,8 @@ public final class CacheAdvance<T: Codable> {
     }
 
     deinit {
-        try? writer.close()
-        try? reader.close()
+        try? writer.closeHandle()
+        try? reader.closeHandle()
```

Comment:
these method renames are due to code that is below. No functional changes. Keep scrolling!